src/TestSuite/Fixture/TestFixture.php

```diff
@@ -264,8 +264,8 @@ protected function _schemaFromReflection()
     /**
      * Gets/Sets the TableSchema instance used by this fixture.
      *
-     * @param \Cake\Database\Schema\TableSchema|\Cake\Database\Schema\TableSchemaInterface|null $schema The table to set.
-     * @return \Cake\Database\Schema\TableSchema|\Cake\Database\Schema\TableSchemaInterface|null
+     * @param \Cake\Database\Schema\TableSchema|null $schema The table to set.
```

Comment:
Same here!

Reply:
Then the docblock should be updated in future when the typhint is changed.

Reply:
Fair enough then..